code/modules/antagonists/gang/abilities/_gang_leader_ability_holder.dm

```diff
@@ -272,5 +272,76 @@
 		antag_role.gang.locker = locker
 
 		M.abilityHolder.removeAbility(/datum/targetable/gang/set_gang_base)
-
+		var/datum/targetable/gang/move_gang_base/newAbil = M.abilityHolder.addAbility(/datum/targetable/gang/move_gang_base)
+		newAbil.doCooldown()
 		return
+
+
+
+/datum/targetable/gang/move_gang_base
```

Comment:
Wait this is just a copy paste of `set_gang_base`, at least make it inherit so you're not duplicating all these checks here (it's already missing the z level check I added to the base one)